Backpack/Rating/Classes/BPKRating.m

```diff
@@ -250,6 +249,16 @@ - (CGSize)systemLayoutSizeFittingSize:(CGSize)targetSize {
 
 - (void)updateStyle {
     self.ratingBubble.ratingValue = self.ratingValue;
+    if (self.ratingValue < 6.00) {
+        self.textWrapper.title = self.title.lowRatingText;
+        self.textWrapper.subtitle = self.subtitle.lowRatingText;
```

Comment:
I wonder if we should move the colouring logic of the RatingBubble to here too, so that it's all in one place.
For example
```
if (self.ratingValue < 6.00) {
  ...
  self.ratingBubble.backgroundColor = BPKColor.panjin;
} else .....
```

Reply:
wdyt?

Reply:
You're right I totally forgot that refactor